middleware/swagger-metadata.js

```diff
@@ -32,7 +32,8 @@ var debug = require('debug')('swagger-tools:middleware:metadata');
 var mHelpers = require('./helpers');
 var multer = require('multer');
 var parseurl = require('parseurl');
-var pathToRegexp = require('path-to-regexp');
+var { pathToRegexp } = require('path-to-regexp');
+var regexEscape = require('regex-escape');
```

Comment:
missing `debug` module, possibly others

Reply:
3 missing in total:
**s-yaml**     😟  PKG ERR!  Not in the package.json. Found in: /lib/specs.js
**debug**       😟  PKG ERR!  Not in the package.json. Found in: /index.js, /middleware/swagger-router.js, /middleware/swagger-metadata.js, /middleware/swagger-validator.js, /middleware/swagger-security.js
**qs**          😟  PKG ERR!  Not in the package.json. Found in: /middleware/helpers.js